azure-pipelines.yml

```diff
@@ -124,16 +131,25 @@ jobs:
   steps:
   - script: ./build.sh -c $(buildConfiguration)
     displayName: 'Build'
+
   - script: ./test.sh -c $(buildConfiguration) -p Unit
     displayName: 'Unit tests'  
+
   - task: DownloadPipelineArtifact@2
     inputs:
       buildType: 'current'
       artifactName: 'testArtifacts'
       targetPath: '$(Build.SourcesDirectory)/artifacts'
+
   - script: ./tools/dotnet-linux/dotnet build -c $(buildConfiguration) ./test/TestAssets/TestAssets.sln
     displayName: 'Build test assets'
+
   - script: ./test.sh -c $(buildConfiguration) -p Acceptance
     displayName: 'Acceptance tests'   
-
-   
+  
+  - task: PublishTestResults@2
```

Comment:
Why there are two same tasks here? The same as above PublishTestResults@2

Reply:
They are two different jobs. One is windows acceptance build, and the other one is for macOS and Linux.